features/features.go

```diff
@@ -31,8 +30,7 @@ const (
 
 // List of features and their default value, protected by fMu
 var features = map[FeatureFlag]bool{
-	unused:                   false,
-	IDNASupport:              false,
+	unused: false,
```

Comment:
hmm, weird `gofmt -l -w features/features.go` didn't align the map, I did it manually.